robottelo/ui/locators/base.py

```diff
@@ -1279,6 +1279,24 @@
         "not(contains(@class, 'disabled'))]/a"
     ),
     "roles.filters.search": (By.XPATH, "//input[@type='search']"),
+    "roles.filters_button": (
+        By.XPATH,
+        "//td/em[normalize-space(.)='%s']"
+        "/following::td//a[@data-id='aid_filters']"
+        " | //td/span/a[normalize-space(.)='%s']"
+        "/following::td//a[@data-id='aid_filters']"),
```

Comment:
`//td[contains(., '%s')]/following-sibling::td//a[@data-id='aid_filters']` works with both styles and doesn't require workarounds like `locators['roles.filters_button'] % (role_name, role_name)` later.